### ISSUE TITLE
new stores and products in spreadsheet

### ISSUE THREAD

**Issue by KonstiSchaetz** [Diconium-WirVsVirus/dds-wvv-backend#25]
Create new store profiles:
- [x] Schwesterherz x 2
- [x] Küchenliebe x 3
- [x] BUCHBOX!
- [x] Peccato Fashion Store x 3
- [x] Prachtmädchen 

Add products for 
- [x] Schwesterherz x 2
- [x] Küchenliebe x 3

Please assign this issue back to me @KonstiSchaetz, when done. 

### Data
https://docs.google.com/spreadsheets/d/1-YY3Ebs6cOpeV5oZw6EOfZWCm0ysyfa1zJ9OuAJ_tR0/edit#gid=0

**Comment by cnasenberg** [comment]
Hi @KonstiSchaetz, do you also have a German description for Schwesterherz & Küchenliebe?

**Comment by schugnic** [comment]
I think the correct space for this topic would be the business space.
This is not development related.

**Comment by cnasenberg** [comment]
Hopefully not for long. Right now we have to add the data to commercetools though.

**Comment by cnasenberg** [comment]
**Please Help**: Did someone change some configuration? I cannot manually add addresses for commerce tools channels anymore for some reason. And I cannot delete channels manually, only through REST.

**Comment by cnasenberg** [comment]
@KonstiSchaetz could you take a look at the data? I left some comments... For example: We should have German descriptions if possible. Currently we are all German based.

**Comment by jonathan-wiens** [comment]
Can we close this?

**Comment by KonstiSchaetz** [comment]
alright, I am going to translate it


Am Sa., 28. März 2020 um 00:16 Uhr schrieb Claudia <notifications@github.com

**Comment by KonstiSchaetz** [comment]
list is updated

**Comment by cnasenberg** [comment]
Hi @KonstiSchaetz , please do not add more stuff to the issue, especially not without notifying me. 😅 Instead, create a new issue with only a limited number of stores to add. Maybe 5 at a time? Then they can be picked up by different people who can work in parallel. Let's talk about it if you like, to find the best process for us.

**Comment by cnasenberg** [comment]
I added/updated all stores listed in the description above. Please review! 

We'll have to create new issues for: 
- Buchhandlung am Rathaus (line 16)
- also: lines 23 - 39!!! (Question: Why not start from line 17?) 
### Data
https://docs.google.com/spreadsheets/d/1-YY3Ebs6cOpeV5oZw6EOfZWCm0ysyfa1zJ9OuAJ_tR0/edit#gid=0

**Comment by cnasenberg** [comment]
Hello @KonstiSchaetz ! Thank you for adding the last products! <3 Are there still any open tasks? If not, let us close this issue. :)
